language/fr.json

```diff
@@ -102,66 +102,67 @@
         },
         {
-          "label": "Couleur du texte d'une cellule (mise en avant)",
-          "description": "Choisir la couleur du texte d'une cellule du tableau quand elle est mise en avant.",
+          "label": "Couleurs de texte des cellules sélectionnées",
+          "description": "Choisir une couleurs de texte pour les cellules sélectionnées.",
           "default": "#ffffff"
         },
         {
-          "label": "Cell id color (highlighted)",
-          "description": "Choose the color of the clue id inside a table cell that is highlighted.",
+          "label": "Couleur des indices numériques des cellules sélectionnées",
+          "description": "Choisir la couleur des indices numériques dans les cellules des cellules sélectionnées",
           "default": "#e0e0e0"
         }
       ]
     },
     {
-      "label": "Contextes comportementaux",
-      "description": "Ces options vous permettront de contrôler le comportement de la tâche.",
+      "label": "Options de comportement",
+      "description": "Ces options permettent de contrôler le comportement de l'activité.",
       "fields": [
         {
           "label": "Nombre de mots à afficher",
-          "description": "Créer un lot aléatoire de mots à partir de l'ensemble. Le lot contiendra toujours au moins tous les mots qui ont été fixés sur la grille quel que soit ce paramètre. Une valeur vide ou 0 signifie qu'il faut utiliser tous les mots."
+          "description": "Crée une sélection aléatoire de mots. Cette sélection va toujours contenir tous les mots qui sont fixés sur la grille, peu importe la valeur de ce paramètre. Une valeur vide ou de 0 signifie qu'il faut d'utiliser tous les mots."
         },
         {
-          "label": "Activer le retour instantané"
+          "label": "Activer le feedback instantané"
         },
         {
-          "label": "Noter les mots",
-          "description": "Si cette option est activée, les mots seront comptés afin de déterminer la note. Sinon, les caractères corrects seront comptés."
+          "label": "Score par mot",
+          "description": "Si cette option est activée, les mots seront comptés pour déterminer le score. Autrement le score sera calculé par caractère."
         },
         {
-          "label": "Appliquer des pénalités",
-          "description": "Si cette option est activée, chaque mauvaise réponse recevra une pénalité de -1."
+          "label": "Appliquer les pénalités",
+          "description": "Si cette option est activée, chaque réponse incorrecte entrainera une pénalité de -1 point au score."
         },
         {
-          "label": "Activer \"Recommencer\""
+          "label": "Activer le bouton \"Recommencer\""
         },
         {
-          "label": "Activer \"Solution\""
+          "label": "Activer le bouton \"Solution\""
         }
       ]
     },
     {
-      "label": "Options et textes",
+      "label": "Interface utilisateur",
+      "importance": "low",
```

Comment:
This field is not required for a translation file, please cmp. https://github.com/otacke/h5p-crossword/blob/master/language/.en.json